examples/experimental/koen/MLP Plan.ipynb

```diff
@@ -118,6 +118,26 @@
   {
```

Comment:
<p>Calling model.send() in the plan feels strange to me. If we're sending the plan to a remote worker and running it there then why would the plan (which is already on the remote machine) send data somewhere else?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>This notebook was experimental and is outdated, please check the Sy Resnet nb</p>